css/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <div>This is a Landing Page</div> 
+      <div><img id="header-img" src="Images/IMG_5372.JPG" alt="Photo of a puppy"/ width="300px"></div>
+      
+      <div><nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#header">Index</a></li>
+          <li><a class="nav-link" href="#video">Video</a></li>
+          <li><a class="nav-link" href="#header">Contact</a></li>
+        </ul>
+      </nav></div>
+
+    </header>
+    
+    <div>
+     <video id="video" src="video/Video de WhatsApp 2023-02-19 a las 18.56.38.mp4" controls alt="Video of a puppy" width="300px"/>
+    </div>
+
+    <div>
+      <form id="form" action="" method="post">
+        <input id="email" type="email" name="email" require placeholder="We need your email"></input>
+        <input id="submit" type="submit">Submit the email</input>
+      </form>
+    </div>
+
+    <script  src="js/index.js"></script>
```

Comment:
remember my first suggestion, please...You are pretty close to making code in a more professional way...good luck with the corrections...see you soon!